virt/virt-2-6-release-notes.adoc

```diff
@@ -32,10 +32,28 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-7544: Microsoft's Windows Server Virtualization Validation Program (SVVP) applies to RHCOS workers
 
+* OpenShift Virtualization is certified in Microsoft’s Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS 8 workers. In the Microsoft SVVP Catalog, they are named Red Hat OpenShift Container Platform 4 on RHEL CoreOS 8.
+
+** Intel and AMD CPUs.
+
 //CNV-9723: VMs are now migrated when a node is drained
 
 //CNV-9521: VMs that use UEFI can now be booted by using secure boot
 
+* You can boot a virtual machine (VM) in Extensible Firmware Interface (EFI) mode.
++
+EFI, like legacy BIOS, initializes hardware components and operating system image files when a computer starts. EFI supports more modern features and customization options than BIOS, enabling faster boot times.
++
+It stores all the information about initialization and startup in a file with a `.efi` extension, which is stored on a special partition called EFI System Partition (ESP). The ESP also contains the boot loader programs for the operating system that is installed on the computer.
```

Comment:
Any specific reason for this paragraph? If not, then I'd propose to drop it.

Reply:
Are you referring to the paragraph starting with "EFI, like legacy BIOS" or the one following it? 

If the 2nd paragraph, I'm fine dropping it. But if you mean the "EFI, like legacy BIOS" paragraph, I feel like that is useful info.

Reply:
"It stores all the information about initialization and startup in a file with a `.efi` extension, which is stored on a special partition called EFI System Partition (ESP). The ESP also contains the boot loader programs for the operating system that is installed on the computer."

this can be removed.

Reply:
Removed unneeded paragraph